notifications/rest/notifications/send-notification-with-badge/send-notification-with-badge.3.x.js

```diff
@@ -12,11 +12,7 @@ const service = client.notify.services('ISXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX');
 service.notifications
   .create({
     identity: '00000001',
-    apn: `{"aps" :'
-            '{ "alert":'
-              '{"title":"Bob alert",'
-               '"body" : "Bob, you just received a badge"},'
-            '"badge" : 1 }}'`,
+    apn: '{"aps" :{ "alert": { "title":"Bob alert", "body" : "Bob, you just received a badge"},"badge" : 1 }}}'
```

Comment:
@devidliik perhaps it will be worth to use JSON.stringify here as it provides more readability to the data. wdyt?